contentstore/models.py

```diff
@@ -92,6 +92,26 @@ class MessageSet(models.Model):
     def __str__(self):
         return "%s" % self.short_name
 
+    def set_max(self, lang):
+        return self.messages.filter(lang=lang).count()
```

Comment:
Possible bikeshed but I worry our future selves would interpret this as a setter (set verb) method with the "lang" as an argument instead of a getter (set noun)